frontend/components/Sidebar.tsx

```diff
@@ -0,0 +1,192 @@
+import { Button } from 'antd';
+import { ReactElement, useMemo, useState } from 'react';
+import {
+  MenuOutlined,
+  UserOutlined,
+  BookOutlined,
+  QuestionCircleOutlined,
+  BarChartOutlined,
+} from '@ant-design/icons';
+import styled from 'styled-components';
+import slideStyle from './slide.module.scss';
+
+const MenuHeading = styled.h1`
```

Comment:
this font size should probably be smaller (but still a little bigger than the MenuElements)

Make sure to make the font size dynamic as well